packages/clerk-js/src/ui/icons/request-auth.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" fill="none"><path fill="gray" fill-rule="evenodd" d="M6 7.6v-.8a2 2 0 0 1 4 0v.8a.8.8 0 0 1 .8.8v2a.8.8 0 0 1-.8.8H6a.8.8 0 0 1-.8-.8v-2a.8.8 0 0 1 .8-.8Zm3.2-.8v.8H6.8v-.8a1.2 1.2 0 0 1 2.4 0Z" clip-rule="evenodd"/><path fill="gray" fill-rule="evenodd" d="M13.074 2.891a7.2 7.2 0 1 0 .021 10.196s.414-.464 0-.884-.886 0-.886 0a5.948 5.948 0 1 1-.02-8.427l-1.042 1.042c-.246.246-.16.442.186.442h2.5c.628 0 .628 0 .628-.628v-2.5c0-.346-.198-.43-.442-.186l-.945.945Z" clip-rule="evenodd"/></svg>
```

Comment:
The `fill` props here should be `currentColor`

Reply:
Thanks Nikos, nice catch